doc/source/serve/advanced-guides/multi-app-container.md

```diff
@@ -0,0 +1,175 @@
+(serve-container-runtime-env-guide)=
+# Run Multiple Applications in Different Containers
+
+This section goes over how you can manage dependencies per application in a more isolated manner by running your Serve applications in separate containers with different images.
+
+:::note
+This is an experimental feature and the API is subject to change. If you have additional feature requests or run into issues, please submit them on [Github](https://github.com/ray-project/ray/issues).
+:::
+
+## Install podman
+
+The `container` runtime environment feature uses [Podman](https://podman.io/) to start and run containers. Please follow these steps to install Podman in your environment. For instance, if you are using Kuberay, you need to use an image that has podman installed for the head and worker group specs.
+
+::::{tab-set}
```

Comment:
Non-blocker: I kinda feel this doesn't need to be a separate tabs. Just link directly to the Podman doc should cover most case. And can probably just add a note for Ubuntu 10.04 or older if we think that's important.

Reply:
Thanks for the suggestion, makes sense!